distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 - empowering the smart home
+Documentation=https://docs.openhab.org
```

Comment:
Err, only the community is https, docs is hosted on github pages, where we cannot do ssl...